frontend/src/components/Confirm/Confirm.styles.ts

```diff
@@ -0,0 +1,88 @@
+import styled from '@emotion/styled';
+import { COLOR } from 'src/constants';
+import Flex from 'src/styles/Flex';
+
+const Wrapper = styled.div<{ isOpened: boolean }>`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  background-color: ${COLOR.BLACK + 'cc'};
```

Comment:
```suggestion
  background-color: ${COLOR.BLACK}cc;
```

이렇게 쓰지 않은 이유가 따로 있을까요?.? 취향차이?!

Reply:
오 생각도 안하고 있었어여